test/common/models/Message.js

```diff
@@ -88,10 +88,10 @@ describe('The Message class', function() {
 
     it('should allow defining optional properties through the opts object', function() {
       var message = new jmap.Message({}, 'id', 'threadId', ['inbox'], {
-        from: {
+        from: [{
```

Comment:
Actually, `from` being an array does make the code use `nthElementOrDefault`, but I'll add more item, no problem.